pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[build-system]
+requires      = ["setuptools>=61.0.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "Autotype"
+version = "1.0.0"
+description = "A quick and small python script that helps you autotype on websites."
+readme = "README.md"
+authors = [{ name = "Tushar Gupta", email = "codingid6@gmail.com" }]
+license = { file = "LICENSE" }
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+]
+keywords = ["Autotype", "simulator", "commandline"]
+dependencies = [
+    "pynput",
```

Comment:
It is better to fix the dependencies' version using `~` or `^` to avoid fails if a dependent dependencies stable version upgrades or does a major release. https://python-poetry.org/docs/dependency-specification/